### ISSUE TITLE
Update Transmission to 2.94

### ISSUE THREAD

**Issue by Anonym-tsk** [vgist/dockerfiles#3]
Please update Transmission to 2.94
[This issue](https://github.com/haugene/docker-transmission-openvpn/issues/510) can help you.
Also you can see [this Dockerfile](https://github.com/linuxserver/docker-transmission/blob/master/Dockerfile), they use flag _--repository http://nl.alpinelinux.org/alpine/edge/main_

**Comment by zgist** [comment]
Thank you, I will wait for the release of Alpine 3.8, and update.

**Comment by Anonym-tsk** [comment]
@zgist and please add PUID and PGID environment variables for running transmission with correct permissions.

**Comment by zgist** [comment]
Sure, I'll add custom permissions tomorrow.

**Comment by zgist** [comment]
https://github.com/vgist/dockerfiles/commit/481d80d7a134b74d10c5827527d3ffa8b5f36af8

custom volume permissions done

**Comment by Anonym-tsk** [comment]
Thank you for your job.
But i created better (imho) [docker image with transmission](https://github.com/Anonym-tsk/docker-transmission). You can merge my code, if you want.

**Comment by zgist** [comment]
You are right, I choose `usermod` `groupmod` command to moodify user, thanks.
